core/src/main/java/edu/wpi/grip/core/Main.java

```diff
@@ -23,22 +21,13 @@
  */
 public class Main {
 
-    @Inject
-    private Project project;
-    @Inject
-    private PipelineRunner pipelineRunner;
-    @Inject
-    private EventBus eventBus;
-    @Inject
-    private Logger logger;
+    @Inject private Project project;
+    @Inject private PipelineRunner pipelineRunner;
+    @Inject private EventBus eventBus;
+    @Inject private Logger logger;
 
     @SuppressWarnings("PMD.SystemPrintln")
     public static void main(String[] args) throws IOException, InterruptedException {
-        // Close GRIP when we get SIGHUP.  This signal is sent, for example, when GRIP is run in an SSH session
-        // and the session is closed.
-        Signal.handle(new Signal("HUP"), signal -> SafeShutdown.exit(0));
```

Comment:
Why is that bad? If you're testing robot code and deploying new GRIP stuff, I don't understand why you would want it to stop when you're disconnected. The reason we used to do that was to avoid letting multiple GRIPs run at the same time, but that's not a problem anymore with the wrapper script